### PR TITLE
Update/jetpack plugin assets mag 16

### PR DESCRIPTION
**Pull request by elliottprogrammer**
WordPress.org allows displaying localized versions of plugin banners, by adding the language code to the existing banner asset URLs. In this PR we add Jetpack banners for all mag-16 languages, except for English and Spanish because they already existed prior.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

Internal reference: p1696242919190359-slack-C01264051NE
Previous PR that added Spanish (ES) banner: https://github.com/Automattic/jetpack/pull/33436

## Does this pull request change what data or activity we track or use?

NO

## Testing instructions:

* Check that the file names are correct.
* Check for typos in the images.

### CODE REVIEW

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-he_HE.png
Comment (jeherve) [inline projects/plugins/jetpack/.w.org-assets/banner-1544x500-he_HE.png:1]: Hebrew is `he_IL` I believe.

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-ar_AR.png
Comment (jeherve) [inline projects/plugins/jetpack/.w.org-assets/banner-1544x500-ar_AR.png:1]: What language would `ar_AR` be?

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-ja_JA.png
Comment (jeherve) [inline projects/plugins/jetpack/.w.org-assets/banner-1544x500-ja_JA.png:1]: Japanese is just `ja` afaik.

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-ko_KO.png
Comment (jeherve) [inline projects/plugins/jetpack/.w.org-assets/banner-1544x500-ko_KO.png:1]: Korean is `ko_KR` I think.

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-pt-br_PT-BR.png
Comment (jeherve) [inline projects/plugins/jetpack/.w.org-assets/banner-1544x500-pt-br_PT-BR.png:1]: This doesn't seem correct. It should have been `pt_BR` I think.

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-sv_SV.png
Comment (jeherve) [inline projects/plugins/jetpack/.w.org-assets/banner-1544x500-sv_SV.png:1]: Swedish is `sv_SE` I think.

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-zh-cn_ZH-CN.png
Comment (jeherve) [inline projects/plugins/jetpack/.w.org-assets/banner-1544x500-zh-cn_ZH-CN.png:1]: Here, just like with Brazilian Portuguese, I think the code is duplicated. I believe it should be `zh_CN`.

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-zh-tw_ZH-TW.png
Comment (jeherve) [inline projects/plugins/jetpack/.w.org-assets/banner-1544x500-zh-tw_ZH-TW.png:1]: Same like above. `zh_TW`

**Review by jeherve — CHANGES_REQUESTED** [review]

**Comment by elliottprogrammer** [comment]
@jeherve, Thanks very much for reviewing!
Hmm, maybe I'm misunderstanding how to format the filenames? 🤔
I was under the impression that the locale slug should be added to the tail end of the filename, in all CAPS, **following an underscore**. So in other words, everything prior to the underscore(_) is just filename and has no bearing on what the locale is.  (Did I interpret this correctly?)

I used the following locale slugs: `'ja', 'fr', 'es', 'pt-br', 'de', 'it', 'nl', 'he', 'zh-cn', 'zh-tw', 'id', 'ru', 'tr', 'sv', 'ar', 'ko'` (These are the slugs we use on Jetpack.com, Calypso, WPCOM forum blog, etc., and I assumed they are the same for the Jetpack plugin, but correct me if I'm wrong, maybe I am mistaken here).

So, from the designer I was given filenames that were named with these slugs, for example, `banner-1544x500-he.png`, `banner-1544x500-ar.png`, `banner-1544x500-pt-br.png`, so then I simply appended an underscore to the name, followed by the locale-slug (in caps), so therefore the 3 filename examples here became, `banner-1544x500-he_HE.png`, `banner-1544x500-ar_AR.png`, `banner-1544x500-pt-br_PT-BR.png`.

Is this not correct? Am I misunderstanding something?
Any help is much appreciated! Thanks! 🙂

**Comment by jeherve** [comment]
> I used the following locale slugs: `'ja', 'fr', 'es', 'pt-br', 'de', 'it', 'nl', 'he', 'zh-cn', 'zh-tw', 'id', 'ru', 'tr', 'sv', 'ar', 'ko'` (These are the slugs we use on Jetpack.com, Calypso, WPCOM forum blog, etc., and I assumed they are the same for the Jetpack plugin, but correct me if I'm wrong, maybe I am mistaken here).

Right, that's the tricky part. WordPress.com Simple uses custom locale slugs, and doesn't bother with country variants iirc.

Self-hosted sites, on the other hand, can have multiple variants for a language, listed here:
https://translate.wordpress.org/
In general, those slugs follow the same format: {language}_{VARIANT}, where {VARIANT} may be a country or a language variant, or may not exist.

As an example, for French, multiple language codes are available:

- French (France) (`fr_FR`)
- French (Belgium) (`fr_BE`)
- French (Canada) (`fr_CA`)

You can check this file for a full list of locales, and you see the `wp_locale` key as an indicator for your work:
https://github.com/GlotPress/GlotPress/blob/5d547417bce54e897e00dc6b45ecd502a0442668/locales/locales.php#L4

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-he_HE.png
Comment (elliottprogrammer) [reply projects/plugins/jetpack/.w.org-assets/banner-1544x500-he_HE.png:1]: Fixed ☑

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-ar_AR.png
Comment (elliottprogrammer) [reply projects/plugins/jetpack/.w.org-assets/banner-1544x500-ar_AR.png:1]: Fixed ☑

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-ja_JA.png
Comment (elliottprogrammer) [reply projects/plugins/jetpack/.w.org-assets/banner-1544x500-ja_JA.png:1]: Fixed ☑

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-ko_KO.png
Comment (elliottprogrammer) [reply projects/plugins/jetpack/.w.org-assets/banner-1544x500-ko_KO.png:1]: Fixed ☑

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-pt-br_PT-BR.png
Comment (elliottprogrammer) [reply projects/plugins/jetpack/.w.org-assets/banner-1544x500-pt-br_PT-BR.png:1]: Fixed ☑

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-sv_SV.png
Comment (elliottprogrammer) [reply projects/plugins/jetpack/.w.org-assets/banner-1544x500-sv_SV.png:1]: Fixed ☑

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-zh-cn_ZH-CN.png
Comment (elliottprogrammer) [reply projects/plugins/jetpack/.w.org-assets/banner-1544x500-zh-cn_ZH-CN.png:1]: Fixed ☑

File: projects/plugins/jetpack/.w.org-assets/banner-1544x500-zh-tw_ZH-TW.png
Comment (elliottprogrammer) [reply projects/plugins/jetpack/.w.org-assets/banner-1544x500-zh-tw_ZH-TW.png:1]: Fixed ☑

**Comment by elliottprogrammer** [comment]
> > I used the following locale slugs: `'ja', 'fr', 'es', 'pt-br', 'de', 'it', 'nl', 'he', 'zh-cn', 'zh-tw', 'id', 'ru', 'tr', 'sv', 'ar', 'ko'` (These are the slugs we use on Jetpack.com, Calypso, WPCOM forum blog, etc., and I assumed they are the same for the Jetpack plugin, but correct me if I'm wrong, maybe I am mistaken here).
> 
> Right, that's the tricky part. WordPress.com Simple uses custom locale slugs, and doesn't bother with country variants iirc.
> 
> Self-hosted sites, on the other hand, can have multiple variants for a language, listed here: https://translate.wordpress.org/

Ahhh... I see now, thanks @jeherve!
Ok excellent, I've renamed the banner filenames to include the correct `wp_locale`'s. They all should be good now. 👍

**Review by jeherve — APPROVED** [review]

**Comment by jeherve** [comment]
I've deployed the banners to the directory, everything seems to look good! 🥳 

- https://wordpress.org/plugins/jetpack/
- https://ar.wordpress.org/plugins/jetpack/
- https://de.wordpress.org/plugins/jetpack/
- https://fr.wordpress.org/plugins/jetpack/
- https://he.wordpress.org/plugins/jetpack/
- https://id.wordpress.org/plugins/jetpack/
- https://ja.wordpress.org/plugins/jetpack/
- https://ko.wordpress.org/plugins/jetpack/
- https://nl.wordpress.org/plugins/jetpack/
- https://br.wordpress.org/plugins/jetpack/
- https://ru.wordpress.org/plugins/jetpack/
- https://sv.wordpress.org/plugins/jetpack/
- https://tr.wordpress.org/plugins/jetpack/
- https://cn.wordpress.org/plugins/jetpack/
- https://tw.wordpress.org/plugins/jetpack/

**Pull request merged**